Resources/ArmTemplates/parameters.json

```diff
@@ -1,105 +1,120 @@
+      }
+    },
+    "appSharedResourceGroup": {
+      "value": "__appSharedResourceGroup__"
+    },
+    "appSharedStorageAccountName": {
+      "value": "__appSharedStorageAccountName__"
+    },
+    "aseHostingEnvironmentName": {
+      "value": "__AseServiceName__"
+    },
+    "aseResourceGroup": {
+      "value": "__aseResourceGroup__"
+    },
+    "aspInstances": {
+      "value": __aspInstances__
+    },
+    "aspTier": {
+      "value": "__aspTier__"
+    },
+    "aspSize": {
+      "value": "__aspSize__"
+    },
+    "apimDSSProxyAddress": {
+      "value": "__apimDSSProxyAddress__"
+    },
+    "AppLoginEmail": {
+      "value": "__AppLoginEmail__"
+    },
+    "AppLoginPassword": {
+      "value": "__AppLoginPassword__"
+    },
+    "cdnUrl": {
+      "value": "__cdnUrl__"
+    },
+    "apimDSSApiKey": {
+      "value": "__apimDSSApiKey__"
+    },
+    "cosmosDBName": {
+      "value": "__appSharedCosmosDBName__"
+    },
+    "cosmosDbKey": {
+      "value": "__appSharedCosmosPrimaryKey__"
+    },
+    "UserSessionsCollection": {
+      "value": "__UserSessionsCollection__"
+    },
+    "ContentCollection": {
+      "value": "__ContentCollection__"
+    },
+    "CosmosDbResourceGroup": {
+      "value": "__appSharedResourceGroup__"
+    },
+    "cosmosDbCollectionName": {
+      "value": "__cosmosDbCollectionName__"
+    },
+    "cosmosContentDbCollectionName": {
+      "value": "__cosmosContentDbCollectionName__"
+    },
+    "actionplanAppDatabaseNeedsCreation": {
+      "value": __CosmosDbDatabaseDoesntExist__
+    },
+    "actionplanAppDatabaseThroughput": {
+      "value": __actionplanAppDatabaseThroughput__
+    },
+    "AuthSettingsIssuer": {
+      "value": "__AuthSettingsIssuer__"
+    },
+    "AuthSettingsClientSecret": {
+      "value": "__AuthSettingsClientSecret__"
+    },
+    "AuthSettingsClientId": {
+      "value": "__AuthSettingsClientId__"
+    },
+    "AuthSettingsAccountEndpoint": {
+      "value": "__AuthSettingsAccountEndpoint__"
+    },
+    "SubscriptionSettingsEndpoint": {
+      "value": "__SubscriptionSettingsEndpoint__"
+    },
+    "SummaryEndpoint": {
+      "value": "__SummaryEndpoint__"
+    },
+    "contentUrl": {
+      "value": "__contentUrl__"
+    },
+    "subscriptionUrl": {
+      "value": "__subscriptionUrl__"
+    },
+    "workspaceName": {
+      "value": "__sharedLogAnalyticsWorkspace__"
+    },
+    "workspaceResourceGroup": {
+      "value": "__sharedLogAnalyticsResourceGroup__"
+    },
+    "CmsTokenEndPointUrl": {
+      "value": "__CmsTokenEndPointUrl__"
+    },
+    "CmsGraphApiUrl": {
+      "value": "__CmsGraphApiUrl__"
+    },
+    "CmsClientId": {
+      "value": "__CmsClientId__"
+    },
+    "CmsClientSecret": {
+      "value": "__CmsClientSecret__"
+    },
+    "CmsRedisCacheConnectionString": {
+      "value": "__CmsRedisCacheConnectionString__"
```

Comment:
If you update '__CmsRedisCacheConnectionString__' to '__RedisCacheConnectionString__',  we should be able to update the release pipeline to use the predefined connection string within the key vault library sets.

Reply:
Sorry,  that should be '__RedisConnectionString__'